### DISCUSSION TITLE
Can't reach http://localhost:8123/auth/register

### DISCUSSION THREAD

**Discussion by rastacalavera** [HiEventsDev/Hi.Events#202] [Q&A]
Hello,
I've followed the steps in the readme and generated the appropriate keys and placed them in the .env file. When I bring the container up, I exclude the -d so that I can easily see the logs during this testing phase. Everything appears to start without issue but I cannot reach the web GUI to login/register. I've tried using the default compose values, changing and changing them to the IP of the machine (I'm running it on a separate machine from my laptop) and nothing seems to work. No errors in the logs, the browser just returns that the webpage isn't responding. I've included       - APP_DEBUG=true in the compose file and it doesn't seem to show anything unusual as well.

All help and ideas to solve this are welcomed!! Thank you :)

**Comment by cannot-hero** [comment]
#201 this may help you

**Reply by rastacalavera** [reply]
Thanks I can't even reach the page though. I'll try re-doing the secret keys and see if that helps at all.

**Reply by rastacalavera** [reply]
Nope.

**Comment by daveearley** [comment]
Hi @rastacalavera,

Do you have anything else running on port 8123?

Also, could you please paste the output of `docker-compose logs all-in-one`.

Thanks

**Comment by rastacalavera** [comment]
It took about an hour or so but eventually the container got running as expected. The logs didn't show anything specific that services were still starting up, so I just had to guess that might be a solution and wait long enough 

can now reach the web ui to register but can't create a user. 

```:8123/api/users/me:1 Failed to load resource: the server responded with a status of 401 (Unauthorized)```

Every time I try to go to the /api route, I get rerouted to https rather than http and hit a generic page from my reverse proxy. I don't know why it's forcing https when I add /api to the end of the url. The environment variables are all http and not https

**Reply by rastacalavera** [reply]
sounds similar to this https://github.com/HiEventsDev/Hi.Events/issues/201#issuecomment-2351934825 but I can't reach the API page. 
![image](https://github.com/user-attachments/assets/1723f3a0-dcb4-4491-9888-467cbe8b5e4e)

**Reply by FourHeadedDev** [reply]
same issue do your got fixed if yes please help me
